core/src/main/java/com/linecorp/armeria/server/file/AbstractBlockingHttpVfs.java

```diff
@@ -75,12 +99,33 @@ public final HttpFile get(
      * @param contentEncoding the desired {@code 'content-encoding'} header value of the file.
      *                        {@code null} to omit the header.
      * @param additionalHeaders the additional HTTP headers to add to the returned {@link HttpFile}.
-     *
      * @return the {@link HttpFile} at the specified {@code path}
+     *
+     * @deprecated Use {@link #blockingGet(Executor, String, Clock, String, HttpHeaders, MediaTypeResolver)}
+     *     instead.
      */
+    @Deprecated
     protected abstract HttpFile blockingGet(Executor fileReadExecutor, String path, Clock clock,
                                             @Nullable String contentEncoding, HttpHeaders additionalHeaders);
```

Comment:
Should we delegate to the `blockingGet()`?
```suggestion
    protected HttpFile blockingGet(Executor fileReadExecutor, String path, Clock clock,
                                   @Nullable String contentEncoding, HttpHeaders additionalHeaders) {
        return blockingGet(..., MediaTypeResolver.ofDefault();                               
    }
```